libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/MQTT_GetIncomingPacketTypeAndLength_harness.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Insert copyright notice
+ */
+
+/**
+ * @file MQTT_GetIncomingPacketTypeAndLength_harness.c
+ * @brief Implements the proof harness for MQTT_GetIncomingPacketTypeAndLength function.
+ */
+#include "mqtt.h"
+
+extern int32_t NetworkInterfaceReceiveStub( NetworkContext_t context,
+                                            void * pBuffer,
+                                            size_t bytesToRecv );
+
+void harness()
+{
+    NetworkContext_t networkContext;
+
+    __CPROVER_assume( networkContext != NULL );
+
+    MQTTPacketInfo_t incomingPacketInfo;
+
+    MQTT_GetIncomingPacketTypeAndLength( NetworkInterfaceReceiveStub,
+                                         networkContext,
+                                         &incomingPacketInfo );
+}
```

Comment:
This is beyond memory safety, but is there anything about the data written to incomingPacketInfo that we could test?  Like lengths or pointers nonnull or something?

Reply:
No there is not, I had comments about it, and i will add them back.